trl/templates/model_card.md

```diff
@@ -0,0 +1,54 @@
+---
+{{ card_data }}
+---
+
+# Model Card for {{ model_name }}
+
+This model is a fine-tuned version of [{{ base_model }}](https://huggingface.co/{{ base_model }}){% if dataset_name %} on the [{{ dataset_name }}](https://huggingface.co/datasets/{{ dataset_name }}) dataset{% endif %}.
+It has been trained using [TRL](https://github.com/huggingface/trl).
+
+## Quick start
+
+```python
+from transformers import pipeline
+
+question = "If you had a time machine, but could only go to the past or the future once and never return, which would you choose and why?"
+generator = pipeline("text-generation", model="{{ hub_model_id }}", device="cuda")
+output = generator([{"role": "user", "content": question}], max_new_tokens=500)[0]
```

Comment:
500 tokens might be a bit slow as the default for LLMs, so maybe we can try this:

```suggestion
output = generator([{"role": "user", "content": question}], max_new_tokens=128, return_full_text=False)[0]
```

Note that `return_full_text=False` will just return the completion which I a bit simpler below

Reply:
I think you don't need for pipeline, let me check

Reply:
![image](https://github.com/user-attachments/assets/5d003b85-fb84-44d3-9276-ab00f5dcbd4d)